bundle/compliance/cis_gcp/rules/cis_1_15/rule.rego

```diff
@@ -5,20 +5,15 @@ import data.compliance.policy.gcp.data_adapter
 import future.keywords.contains
 import future.keywords.if
 
-default is_valid = false
-
 duration = sprintf("%dh", [90 * 24]) # 90 days converted to hours
 
-is_valid if {
-	date := time.parse_rfc3339_ns(data_adapter.resource.data.createTime)
-	common.date_within_duration(date, duration)
-}
-
 finding = result if {
 	data_adapter.is_api_key
 
+	date := time.parse_rfc3339_ns(data_adapter.resource.data.createTime)
```

Comment:
What will happen if `createTime` is undefined, I think that we'll get out of scope and this line will act as a filter, no?

Reply:
yeah, good catch. changed to `key_created_within_last_90_days`. unlikely though